pkg/subctl/cmd/show_connections.go

```diff
@@ -36,30 +36,35 @@ func getConnectionsStatus(config *rest.Config) []connectionStatus {
 	submarinerClient, err := submarinerclientset.NewForConfig(config)
 	exitOnError("Unable to get the Submariner client", err)
 
+	SubmarinerInstalled = true
 	var status []connectionStatus
 
 	existingCfg, err := submarinerClient.SubmarinerV1alpha1().Submariners(OperatorNamespace).Get(submarinercr.SubmarinerName, v1.GetOptions{})
 	if err != nil {
-		exitOnError("error reading from submariner client", err)
+		if ! SubmarinerNotInstalled(err) {
+			exitOnError("Error obtaining the Submariner resource", err)
+		}
 	}
 
-	gateways := existingCfg.Status.Gateways
-	if gateways == nil {
-		exitWithErrorMsg("No endpoints found")
-	}
+	if SubmarinerInstalled == true {
```

Comment:
It would be more readable if you change to:
`if SubmarinerInstalled == false {
    return status
  }`
  
and then continue the rest of the code.
Otherwise looking good